crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -1123,9 +1120,19 @@ impl actor::HcP2p for HolochainP2pActor {
 
             let byte_count = zome_call_params_serialized.0.len();
 
+            let agent_id = to_agent.to_k2_agent();
+            if space
+                .local_agent_store()
+                .is_agent_local(agent_id.clone())
```

Comment:
Make sure we can't call_remote to ourselves

Reply:
I take it back, one test was accidentally expecting send to self to work. Which it was because it was using the mem transport rather than tx5.